package.json

```diff
@@ -104,8 +104,8 @@
     "remark-validate-links": "^6.0.0"
   },
   "lint-staged": {
-    "src/!(mochitest)**/*.js": [
-      "prettier --write --trailing-comma=es5 --bracket-spacing=true",
+    "src/*/!(mochitest)**/*.js": [
```

Comment:
i've tried a couple of patterns, any removing the mochitest leaves out some files. should we just do every thing i.e `src/**/*.js` ?